src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,78 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
```

Comment:
Actually, what's the purpose of Record<string,never> here instead of React.FC which is by default `React.FC<{}> `

Reply:
To limit property keys to a particular type I think

Reply:
hem, actually I think itu suggestion dari linter but not necessary